chapters/handling-data.tex

```diff
@@ -83,54 +109,86 @@ \subsection{Research replicability}
 Therefore, your code should be written neatly with clear instructions and published openly.
 It should be easy to read and understand in terms of structure, style, and syntax.
 Finally, the corresponding dataset should be openly accessible
-unless for legal or ethical reasons it cannot be.\sidenote{\url{https://dimewiki.worldbank.org/wiki/Publishing_Data}}
+unless for legal or ethical reasons it cannot be.\sidenote{
+  \url{https://dimewiki.worldbank.org/wiki/Publishing_Data}}
+
+Reproducibility and transparency are not binary concepts:
```

Comment:
We are talking about _transparency_ here before we introduce that in the next section. I like this paragraph but I think it should be moved to the end of the _Research transparency_ or _Research credibility_ section.

Reply:
I removed this paragraph in 35f4599c8eafb5b0cbd507ed8054f62a2290c46f because it is repeated from our linked blogpost and I don't think it adds anything by being reproduced here.

Reply:
did you reference the wrong commit?

Reply:
Yes, should be 0de5565432c7b1b6d81b0d203a0dfc8f0a3adce4

Reply:
Cant we just move it? I like that paragraph.

Reply:
I do too but I think it would need to be rewritten since it is already published content, no?

Reply:
ok, I see. Lets remove then.